crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -0,0 +1,539 @@
+    let refs: Vec<(Vec<Gas>, ObjectRef)> = gas
+        .iter()
+        .zip(transfer_objects.iter())
+        .map(|(g, t)| {
+            (
+                g.iter()
+                    .map(|x| (x.compute_object_reference(), x.owner))
+                    .collect(),
+                t.compute_object_reference(),
+            )
+        })
+        .collect();
+    let mut flattened: Vec<Object> = gas.into_iter().flatten().collect();
+    flattened.append(&mut transfer_objects);
+
+    // Setup the network
+    let _ = spawn_test_authorities(flattened.clone(), &configs).await;
+    let clients = test_authority_aggregator(&configs);
+
+    (
+        refs.iter()
+            .map(|(g, t)| {
+                let from = owner;
+                let (_, to) = *g
+                    .iter()
+                    .find(|x| x.1.get_owner_address().unwrap() != from)
+                    .unwrap();
+                Payload::TransferObjectTxPayload((
+                    *t,
+                    (from, to.get_owner_address().unwrap()),
+                    g.clone(),
+                ))
+            })
+            .collect(),
+        accounts,
+        clients,
+    )
+}
+
+async fn init_shared_counter_benchmark(
+    count: usize,
+    configs: NetworkConfig,
+) -> (
+    Vec<Payload>,
+    Arc<HashMap<SuiAddress, KeyPair>>,
+    AuthorityAggregator<NetworkAuthorityClient>,
+) {
+    // create enough gas
+    let mut gas = vec![];
+    let mut counters_gas = generate_gas_objects(count);
+    let publish_module_gas = generate_gas_object();
+    gas.append(&mut counters_gas);
+    gas.push(publish_module_gas.clone());
+
+    // Setup the network
+    let _ = spawn_test_authorities(gas.clone(), &configs).await;
+    let clients = test_authority_aggregator(&configs);
+    let quorum_driver_handler = QuorumDriverHandler::new(clients.clone());
+
+    // Publish basics package
+    eprintln!("Publishing basics package");
+    let package_ref = publish_counter_package(publish_module_gas, configs.validator_set()).await;
+    gas.pop();
+
+    let qd_and_gas = gas
+        .into_iter()
+        .map(|g| (quorum_driver_handler.clone_quorum_driver(), g));
+
+    // create counters
+    let futures = qd_and_gas.map(|(qd, gas_object)| async move {
+        let tx =
+            make_counter_create_transaction(gas_object.compute_object_reference(), package_ref);
+        if let ExecuteTransactionResponse::EffectsCert(result) = qd
+            .execute_transaction(ExecuteTransactionRequest {
+                transaction: tx,
+                request_type: ExecuteTransactionRequestType::WaitForEffectsCert,
+            })
+            .await
+            .unwrap()
+        {
+            let (_, effects) = *result;
+            Payload::SharedCounterTxPayload((
+                (package_ref, effects.effects.created[0].0 .0),
+                effects.effects.gas_object,
+            ))
+        } else {
+            unreachable!();
```

Comment:
I think if you really don't expect to hit this, lets use a call to `panic!` with a description message instead of unreachable